modules/ui/map3d_viewer.js

```diff
@@ -272,6 +275,33 @@ export function uiMap3dViewer(context) {
     map.on('draw', () => redraw());
     map.on('move', () => redraw());
     context.keybinding().on([uiCmd('⌘' + l10n.t('background.3dmap.key'))], toggle);
+  /**
+   * _hashchange
+   * Respond to any changes appearing in the url hash
+   */
+    function _hashchange(){
+      if (context.systems.urlhash?.getParam('3d') === 'true'){
```

Comment:
Let's actually make this parameter check even more lax- just check to see if `map3d` exists as a hash param. it can be equal to anything, or nothing at all. 

i.e. 

`https://rapideditor.org/#map3d`

`http://127.0.0.1:8080/#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=20.16/41.37239/-74.69426&map3d=whatever`

should both work.

Reply:
If the 3d map shouldn't be displayed, it should not be a parameter in the URL- we can just remove it, rather than toggling it from true to false.

Reply:
I've decided to leave it as `map3d=true` since `urlhash.setParam()` must take a key/value pair as its arguments. Also, `map3d=true` will disappear when setParam is called as `urlhash.setParam('map3d', null);